src/async-status-indicator/examples/default.js

```diff
@@ -0,0 +1,51 @@
+const labels = {
+  success: 'Success',
+  error: 'Error',
+  ready: 'Ready',
+  processing: 'Processing'
+};
+
+const icons = {
+  success: <Icon icon='check2' />,
+  error: <Icon icon='x' />,
+  processing: <LoadingSpinner size='1em' overlayColor='transparent' />
+}
+
+class Example extends React.Component {
+
+  state = { state: 'ready' }
+
+  onClick = () => {
+    if (this.timeout) {
```

Comment:
should be `|| timeout2 || timeout3` ?
Or better, just use a single `this.timeout` for every step